src/create-elements.js

```diff
@@ -0,0 +1,95 @@
+import { FRAGMENT, btnRotate } from "./variables.js";
+
+export function createDiv(element, classOfElement, attributeFirst) {
+  let div = document.createElement(element);
+  div.classList.add(classOfElement);
+  if (attributeFirst != undefined) {
```

Comment:
```suggestion
  if (attributeFirst) {
```
Do you really need to check if it is "undefined"?